python/tests/integration/arcticdb/test_update.py

```diff
@@ -245,6 +245,13 @@ def test_my_test(custom_library):
         pass
 
 
+def get_metadata_safe() -> str:
```

Comment:
Just `random_metadata()`  or something would be a fine name, the `safe` in this name is confusing